packages/codegen/src/generateFlushDatabaseFunction.ts

```diff
@@ -0,0 +1,30 @@
+import { Client } from "pg";
+import { Db } from "pg-structure";
+import { Config } from "./config";
+import { isEntityTable, isJoinTable } from "./utils";
+
+/** Creates a `flush_database` stored procedure to truncate all of the tables between tests. */
+export async function createFlushDbFunction(db: Db, client: Client, config: Config): Promise<void> {
```

Comment:
Nothing changed, just moved in from `joist-migration-utils`.

Actually another reason this makes sense is b/c users who bring-their-own-migration library (so don't need/want `joist-migration-utils`) would still want to use `flush_database`...